sys/include/div.h

```diff
@@ -33,17 +33,52 @@ extern "C" {
 /**
  * @brief Approximation of (2**l)/d for d=15625, l=12, 32 bits
  */
-#define DIV_H_INV_15625_32    0x431bde83ul
+#define DIV_H_INV_15625_32     0x431bde83ul
 
 /**
  * @brief Approximation of (2**l)/d for d=15625, l=12, 64 bits
  */
-#define DIV_H_INV_15625_64    0x431bde82d7b634dbull
+#define DIV_H_INV_15625_64     0x431bde82d7b634dbull
+
+/**
+ * @brief Approximation of (2**l)/d for d=625, l=8, 32 bits
+ */
+#define DIV_H_INV_625_32       0x68db8badul
+
+/**
+ * @brief Approximation of (2**l)/d for d=625, l=8, 64 bits
+ */
+#define DIV_H_INV_625_64       0x68db8bac710cb296ull
+
+/**
+ * @brief Approximation of (2**l)/d for d=576, l=0, 32 bits
+ */
+#define DIV_H_INV_576_32       0x71c71dul
+
+/**
+ * @brief Approximation of (2**l)/d for d=576, l=0, 64 bits
+ */
+#define DIV_H_INV_576_64       0x71c71bff8e38e5ull
 
 /**
  * @brief Required shifts for division by 15625, l above
  */
-#define DIV_H_INV_15625_SHIFT 12
+#define DIV_H_INV_15625_SHIFT  12
+
+/**
+ * @brief Required shifts for division by 625, l above
+ */
+#define DIV_H_INV_625_SHIFT    8
+
+/**
+ * @brief Operand for 576/625
+ */
+#define MUL_576                576ul
+
+/**
+ * @brief Operand for 625/576
+ */
+#define MUL_625                625ul
```

Comment:
It was to avoid the "ul" everywhere (8 and 16 bit requirement), but I can remove it.